nemo_text_processing/text_normalization/zh/case.tsv

```diff
@@ -0,0 +1,47 @@
+苹果ＣＥＯ宣布发布新ＩＰＨＯＮＥ
```

Comment:
What is the role of this file?

Reply:
this is the test cases supported now.This case.tav cover all the different cases now.

Reply:
Could please reformat them like [here](https://github.com/NVIDIA/NeMo/tree/main/tests/nemo_text_processing)?